### ISSUE TITLE
Cannot compile CLI: “error: not found: value Universal”

### ISSUE THREAD

**Issue by ProgVal** [Spirals-Team/powerapi#79]
```
stagem2@misc:~/powerapi/powerapi-cli$ ~/sbt/bin/sbt publishLocal
/home/stagem2/powerapi/powerapi-cli/build.sbt:3: error: not found: value Universal
mappings in Universal += downloadBluecove.value -> s"lib/${downloadBluecove.value.name}"
            ^
[error] Type error in expression
```

Scala version: 2.9.2+dfsg-2, from Debian 8 (Jessie)
Java version: 1.7.0_95 (openjdk 7u95-2.6.4-1~deb8u1), from Debian 8
sbt version: 0.13.9 (the latest from http://www.scala-sbt.org/)

**Comment by mcolmant** [comment]
Which branch did you clone?
Works on the master branch. However, you could use `universal:packageZipTarball` to generate an archive, or [download](https://github.com/Spirals-Team/powerapi/releases/download/3.4/powerapi-cli-3.4.tgz) it.

**Comment by ProgVal** [comment]
I cloned the master branch.

And `~/sbt/bin/sbt universal:packageZipTarball` gives the same error as above

**Comment by mcolmant** [comment]
Is up-to-date with the repository? I cannot reproduce the bug with a fresh clone.

**Comment by ProgVal** [comment]
Yes.

Could it be caused by a missing dependency?

**Comment by mcolmant** [comment]
Ok, I think I get it. It's an sbt submodule, then you have to activate sbt in the root folder, and then navigate to the cli project. You can easily do that with:
```
:/powerapi$ sbt "project powerapi-cli" universal:packageZipTarball
```

**Comment by ProgVal** [comment]
Indeed, that worked

**Comment by mcolmant** [comment]
Closes #79
